src/components/dropdownSearch/DropdownSearchReducers.tsx

```diff
@@ -83,23 +125,24 @@ export const dropdownSearchReducer = (state: IDropdownSearchState = dropdownSear
       return {
         ...state,
         id: action.payload.id,
-        selectedOption: action.payload.selectedOption,
+        selectedOptions: state.selectedOptions.push(action.payload.addedSelectedOption),
         isOpened: false,
         activeOption: undefined,
         setFocusOnDropdownButton: false,
       };
     case DropdownSearchActions.add:
       return {
+        ...state,
         id: action.payload.id,
         options: action.payload.optionsDropdown,
-        selectedOption: action.payload.selectedOption,
+        selectedOptions: new FixedQueue<IDropdownOption>([defaultSelectedOption], 1),
```

Comment:
There is a default option variable in the reducer to push a default display value in the dropdown. That was how @pochretien implemented the first DropdownSearch